tsup.config.ts

```diff
@@ -1,23 +1,28 @@
 import { defineConfig } from "tsup";
 
 export default defineConfig({
-  entry: [
-    "src/const.ts",
-    "src/types.ts",
-    "src/urls.ts",
-    "src/headers.ts",
-    "src/saleor-app.ts",
-    "src/verify-jwt.ts",
-    "src/verify-signature.ts",
-    "src/APL/index.ts",
-    "src/APL/redis/index.ts",
-    "src/APL/vercel-kv/index.ts",
-    "src/app-bridge/index.ts",
-    "src/app-bridge/next/index.ts",
-    "src/handlers/next/index.ts",
-    "src/middleware/index.ts",
-    "src/settings-manager/index.ts",
-  ],
+  entry: {
+    const: "src/const.ts",
+    types: "src/types.ts",
+    urls: "src/urls.ts",
+    headers: "src/headers.ts",
+    "saleor-app": "src/saleor-app.ts",
+    "verify-jwt": "src/verify-jwt.ts",
+    "verify-signature": "src/verify-signature.ts",
+    "APL/index": "src/APL/index.ts",
+    "APL/redis/index": "src/APL/redis/index.ts",
+    "APL/vercel-kv/index": "src/APL/vercel-kv/index.ts",
+    "app-bridge/index": "src/app-bridge/index.ts",
+    "app-bridge/next/index": "src/app-bridge/next/index.ts",
+    "settings-manager/index": "src/settings-manager/index.ts",
+    "handlers/shared/index": "src/handlers/shared/index.ts",
+
+    // Deprecated
+    "middleware/index": "src/middleware/index.ts",
+
+    // Mapped exports
+    "handlers/next/index": "src/handlers/platforms/next/index.ts",
+  },
```

Comment:
What does mean this api change - from array to object? Should keys include `/index`?

Reply:
Yes, otherwise we woudlnt' be able to do mapped exports in tsup

So for example:
```
[
  "src/headers.ts"
]
```

is the same as:

```
{
  headers: "src/headers.ts"
}
```